lib/src/customer_io.dart

```diff
@@ -134,8 +134,8 @@ class CustomerIOSDK extends ChangeNotifier {
             _ConfigurationKey.siteId, configurations.siteId);
         await prefs.setOrRemoveString(
             _ConfigurationKey.apiKey, configurations.apiKey);
-        await prefs.setOrRemoveString(
-            _ConfigurationKey.organizationId, configurations.organizationId);
+        await prefs.setOrRemoveBool(
+            _ConfigurationKey.enableInApp, configurations.enableInApp);
```

Comment:
This 1 change here makes me think that this PR shouldn't be a `chore:` but a `fix:`. 

Since `chore:` was used, this change will not get deployed to customers.

Reply:
Discussed this in slack.